charts/nexus-repository-manager/README.md

```diff
@@ -1,10 +1,10 @@
-# Nexus
+# Nexus Repository
 
-[Nexus OSS](https://www.sonatype.com/nexus-repository-oss) provides universal support for all major build tools.
+[Nexus Repository OSS](https://www.sonatype.com/nexus-repository-oss) provides universal support for all major build tools.
 
-- Store and distribute Maven/Java, npm, NuGet, Helm, Docker, P2, OBR, APT, GO, R, Conan components and more.
+- Store and distribute Maven/Java, npm, NuGet, Helm, Docker, p2, OBR, APT, Go, R, Conan components and more.
```

Comment:
NOTE: Wonder if this should read "and more components"...as is reads like you can store components and other stuff, which I don't think is true...but anyway, I didn't notice first pass so leaving alone, for future consideration maybe.